.travis.yml

```diff
@@ -1,35 +1,38 @@
-sudo: false
+sudo:
+  - false
 
-language: cpp
+language:
+  - cpp
 
 cache:
   directories:
-    - odb
+    - linux_dependencies
+    - osx_dependencies
 
 matrix:
   include:
 
-    # Clang 3.9
-    - compiler: clang
-      env: C_COMPILER=clang-3.9 CXX_COMPILER=clang++-3.9
-      addons:
-        apt:
-          sources:
-            - boost-latest
-            - ubuntu-toolchain-r-test
-            - llvm-toolchain-trusty-3.9
-          packages:
-            - clang-3.9
-            - libboost1.55-dev
-            - libboost-filesystem1.55-dev
-            - libboost-iostreams1.55-dev
-            - libboost-program-options1.55-dev
-            - libboost-regex1.55-dev
-            - libboost-log1.55-dev
-            - ninja-build
+      # OS X
+    - os: osx
+      osx_image: xcode9.3
+      env: C_COMPILER=clang CXX_COMPILER=clang++
+      before_install:
+        - brew install ninja
+        - brew install boost@1.55
+        - brew link --overwrite --force boost@1.55
```

Comment:
This old version of Boost is specified because that is the latest supported on Travis Linux machines. But for Mac OS X I think it's better to stick to the default too, instead of reverting to an older one.

Reply:
hmm that too is fine . just it is showing some warnings as you stated in last review.